react-carousel-component/my-app/src/Carousel.js

```diff
@@ -0,0 +1,94 @@
+import { FaRegCircle, FaLessThan, FaGreaterThan } from 'react-icons/fa'
+import { useState, useEffect} from "react";
+import './Carousel.css'
+// import readItems from './PokemonList';
+
+
+// readItems();
+
+export default function Carousel({url, name, characterList}) {
+  const [current, setCurrent] = useState(0)
+
+  const carouselMovement = () => {
+    if (current === characterList.length - 1) {
+      return setCurrent(0)
+    }
+    return setCurrent(current + 1)
+  }
+
+  function handleClick(number) {
+    if (number !== current) {
```

Comment:
This check isn't actually necessary since React will do the check for you. If you call a state setter function and pass the same value, it's a no-op -> nothing happens.